templates/zerver/api/get-user-groups.md

```diff
@@ -0,0 +1,52 @@
+# Get user groups
+
+Get all of the user groups in the realm.
+
+`GET {{ api_url }}/v1/user_groups`
+
+## Usage examples
+
+<div class="code-section" markdown="1">
+<ul class="nav">
+<li data-language="python">Python</li>
+<li data-language="curl">curl</li>
+</ul>
+<div class="blocks">
+
+<div data-language="curl" markdown="1">
+
+```
+curl {{ api_url }}/v1/user_groups \
+    -u BOT_EMAIL_ADDRESS:BOT_API_KEY
+```
+
+</div>
+
+<div data-language="python" markdown="1">
+
+{generate_code_example(python)|/user_groups:get|example}
+
+</div>
+
+</div>
+
+</div>
+
+## Arguments
+
+This endpoint takes no arguments.
```

Comment:
You can replace this with:

    {generate_api_arguments_table|zulip.yaml|/user_groups:get}

And it will automatically point out that the endpoint doesn't take arguments.

Reply:
@YagoGG Thanks! Replaced it.